ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/PostgresDAO.scala

```diff
@@ -767,7 +768,7 @@ case class JDBCPostgresDAO(jdbcUrl: String) extends PostgresDAO with AutoCloseab
 
     offset.foreach { existingOffset =>
       List(1, 2, 3, 4, 5, 6, 7, 8, 9, 10).foreach(
-        preparedDeleteIngestionOverspillEntries.setBytes(_, existingOffset.toByteArray)
+        preparedDeleteIngestionOverspillEntries.setString(_, existingOffset.toHexString)
```

Comment:
@mziolekda, @kamil-da: On review, I'm slightly less comfortable with relying on `HexStringModuleImpl#encode` to never ever change in the future. It'll be slightly more future proof to extract the specific serialization into a helper function in the DB access layer. This can be done in a separate PR though.

cc @remyhaemmerle-da, @moritzkiefer-da:

Reply:
@gerolf-da @cocreature I don't have strong feelings against using `HexStringModuleImpl` either.

@gerolf-da Is there anything stopping us from merging this once PR checks complete? Any benchmarks?

Reply:
No blocker to merge this PR. We can change that, if we deem it necessary, in a subsequent PR. I can also not approve it because I opened it ;)

Reply:
I understand, thanks!

Reply:
Your choice.